src/main/java/no/nav/pto/veilarbportefolje/aktiviteter/KafkaAktivitetMelding.java

```diff
@@ -38,6 +38,7 @@
         BEHANDLING,
         MOTE,
         SAMTALEREFERAT,
+        UTDANNINGAKTIVITET
```

Comment:
Tror ikke at denne endringen ikke skal være med

Reply:
Yep, it will be removed soon